plugin-locationcomponent/src/test/java/com/mapbox/maps/plugin/locationcomponent/LocationPuckManagerTest.kt

```diff
@@ -255,7 +255,7 @@ class LocationPuckManagerTest {
     locationPuckManager.styleScaling(settings)
     verify { locationLayerRenderer.styleScaling(capture(valueSlot)) }
     assertEquals(
-      "[interpolate, [exponential, 0.5], [zoom], 0.0, [literal, [1.0, 1.0, 1.0]], 0.0, [literal, [1.0, 1.0, 1.0]]]",
+      "[interpolate, [exponential, 0.5], [zoom], 0.5, [literal, [2965820.800757861, 2965820.800757861, 2965820.800757861]], 22.0, [literal, [1.0, 1.0, 1.0]]]",
```

Comment:
Curious where are these big numbers come from?

Reply:
@Chaoba as we use constant MAX_ZOOM and MIN_ZOOM now, these values comes from the calculation of the scale expression using `2.0.pow(maxZoom - minZoom)`.

Reply:
Could we also precalculate the `2.0.pow(maxZoom - minZoom)` and put to the constants? 
Suppose no need to calculate every time if vars are constant

Reply:
yeah will do that 👍